src/Common/src/Interop/Windows/kernelbase/Interop.OpenCommPort.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class KernelBase
+    {
+#pragma warning disable BCL0015 // Disable Pinvoke analyzer errors, needed until buildtools can be updated to remove kernel32.dll!OpenCommPort
```

Comment:
This is not present in master it can only be removed from release/UWP6.0 if build tools are updated but it is a good idea to track so if there is a question when merging branches we can make the right choice.